src/CHANGES.txt

```diff
@@ -19,6 +19,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Change the default for AppendENVPath to delete_existing=0, so path
       order will not be changed, unless explicitly set (Issue #3276)
     - Fixed bug which threw error when running SCons on windows system with no MSVC installed.
+    - Update link tool to convert target to node before accessing node member
+    - Update mingw tool to remove MSVC like nologo CCFLAG
+    - Update lex tool to work in mingw environments
```

Comment:
Add that it's going to search for win_flex on windows? (And that it's given preference.

Reply:
added more info to CHANGES.txt in 362aae2